lib/licensee/project_file.rb

```diff
@@ -12,7 +12,10 @@ class File
 
       def initialize(content, filename = nil)
         @content = content
-        @content.encode!(ENCODING, ENCODING_OPTIONS)
+        @content.force_encoding(ENCODING)
+        unless @content.valid_encoding?
```

Comment:
> `:replace:` will remove all the invalid characters altogether from the License

Yes, that's exactly what was causing #79.

Trying force_encoding first, and only running encode (which does above) if that doesn't produce a valid encoding should avoid the latter in many cases. (I didn't realize you could check for a valid encoding, that was the hint I needed.)